### ISSUE TITLE
LILYGO T-Display S3 - no success flashing to 1.6.4/1.7.0

### ISSUE THREAD

**Issue by sany3001** [BitMaker-hub/NerdMiner_v2#477]
I am able to flash my new device to 1.6.3. It starts well. When trying 1.6.4 / 1.7.0 I see blank screen only. Is it known problem?

**Comment by GeeBeeCryptos** [comment]
Same problem with my T-Display S3 !
![image](https://github.com/user-attachments/assets/977504e7-eea0-4da6-910b-b3e19969c219)

**Comment by sany3001** [comment]
See title for my device: LILYGO T-Display S3

**Comment by BitMaker-hub** [comment]
hey, not a known issue. very strange. 
This is just the most common device. So should be good :(
I will check this

**Comment by julio-matarranz** [comment]
Maybe it's a new device like AMOLED V2 ?

**Comment by julio-matarranz** [comment]
What does the monitoring says when your board is plugged ?

**Comment by GeeBeeCryptos** [comment]
New ??? Had it, since at least 9 months. And works like a charme on 1.6.3
Thank you

**Comment by sany3001** [comment]
My old miner are HW 1_2. Here is boot log from HW VL2:

![Uploading IMG_8202.jpeg…]()

**Comment by sany3001** [comment]
Same problem with builds 1.6.4/1.7.0 from GitHub with an old device.

**Comment by TylonHH** [comment]
Wich model do you select in the prerelease?

**Comment by TylonHH** [comment]
Try this: https://github.com/BitMaker-hub/NerdMiner_v2/issues/517

**Comment by soenkegissel** [comment]
**Lilygo T-Display S3**

I can confirm not being able to flash 1.7.0 firmware with non of the firmwares available at https://flasher.bitronics.store/. 
Just black screen after flashing.

But 1.6.4 **NerdminerV2** firmware is working with my Lilygo T-Display S3. I flashed with https://flasher.bitronics.store/

**Comment by TylonHH** [comment]
You tried my workaround?

**Comment by soenkegissel** [comment]
Yes I did. Not no success. Same behavior. Not able to flash 1.7.0.
But I am satisfied with 1.6.4 if it keeps running.
